### PR TITLE
Ce 521/crypto as independent library

### PR DESCRIPTION
**Pull request by shaileshp0110**

### CODE REVIEW

File: dependencies.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  strictVisibility: true
+  transitivity: exports # runtime_deps
+  versionConflictPolicy: highest
+
+dependencies:
+
+
+
+  com.github.pureconfig:
+    pureconfig:
```
Comment (AlexITC) [inline dependencies.yaml]: My understanding is that pureconfig, nor typesafeconfig are required.

Comment (AlexITC) [reply dependencies.yaml]: Now I see that pureconfig is actually used to auto-derive codecs for the crypto classes, I still believe typesafe config is not required.

Comment (shaileshp0110) [reply dependencies.yaml]: I will check this

**Comment by shaileshp0110** [comment]
> Library will need to be publishable from the get go (even if only to a local repo), so we need a solution. Happy to punt that to another story if necessary, but we should not forget.

Yes we have a story in the epic public to the repo ? What do you mean by local repo if its maven repo then I don't think that will be a quick fix.

**Pull request merged**